packages/firestore/exp/src/api/database.ts

```diff
@@ -325,7 +337,7 @@ export function terminate(
 }
 
 function verifyNotInitialized(firestore: Firestore): void {
-  if (firestore._initialized) {
+  if (firestore._initialized || firestore._terminated) {
```

Comment:
Question: what's the reason to check for `_terminated`? Can an instance be terminated without being initialized? If yes, consider adding this bit to the error message below.

Reply:
If a user calls "terminate()" on an uninitialized Firestore, we should also not initialize it. The error message is actually shared between the new SDK and the old SDK and hence, I would rather not touch it.